src/main/java/no/unit/nva/model/PublicationType.java

```diff
@@ -8,7 +8,7 @@
 
 public enum PublicationType {
 
-    JOURNAL_ARTICLE("JournalArticle");
+    JOURNAL_ARTICLE("Article");
```

Comment:
Why is this just Article now?

Reply:
Heavy handed refactoring, methinks, reverting.

Reply:
True. Maybe in the future we will support proceedings from conferences as well. NTNU is using conference proceedings as well